docs/documentation/concepts/things.md

```diff
@@ -61,7 +63,7 @@ Statuses can only have some of the given details. The allowed combinations are l
 
```

Comment:
Apart from NONE, every detail belongs exactly to one status. Could you maybe merge the two tables "Status Details" and "Status Detail Mapping" into a single table? I think the split does not make much sense (I know, not your fault, but it might be a good time to clean it up ;-))